spec/javascript/components/components.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import { Spinner } from 'patternfly-react';
+
+describe('<Spinner/>', () => {
+  it('renders three <Foo /> components', () => {
+    const wrapper = shallow(<Spinner loading />);
+    expect(wrapper.find('.spinner')).to.have.length(1);
```

Comment:
So this is basically one the most annoying things about testing react components. Hopefully you can use snapshot testing with Jest https://jestjs.io/docs/en/snapshot-testing (also i would recommend using [enzime-to-json](https://github.com/adriantoine/enzyme-to-json) instead of the react-renderer in the example).